eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/SectionNavItem.js

```diff
@@ -13,6 +13,25 @@ const StyledSectionNavItem = styled.li`
   display: block;
 `;
 
+const SectionNameOuter = styled.span`
+  padding-left: 0.5em;
+`;
+
+const StyledSectionUpper = styled.div`
+  display: block;
+  padding-top: 1px;
+  margin-left: 0.75em;
+  border-top: 1px solid grey;
+`;
+
+const StyledSectionLower = styled.div`
+  display: block;
+  padding-bottom: 1px;
+  margin-bottom: 1px;
+  margin-left: 0.75em;
+  border-bottom: 1px solid grey;
```

Comment:
All looks good to me. Only one question, does the border-bottom color need to be a variable?

Reply:
Spot on !!